internal/render/node_test.go

```diff
@@ -25,8 +25,27 @@ func TestNodeRender(t *testing.T) {
 	assert.Nil(t, err)
 
 	assert.Equal(t, "minikube", r.ID)
-	e := model1.Fields{"minikube", "Ready", "master", "amd64", "0", "v1.15.2", "Buildroot 2018.05.3", "4.15.0", "192.168.64.107", "<none>", "0", "10", "20", "0", "0", "4000", "7874"}
-	assert.Equal(t, e, r.Fields[:17])
+	e := model1.Fields{
```

Comment:
Might be a good idea to test the taint list is ok. Or at the very least test `formatTaints`.